bcipy/helpers/load.py

```diff
@@ -1,56 +1,49 @@
+from datetime import datetime
 from tkinter import Tk
 import numpy as np
 import pandas as pd
 import logging
 from codecs import open as codecsopen
 from json import load as jsonload
+from shutil import copyfile
+from pathlib import Path
 import pickle
+from bcipy.helpers.parameters import Parameters, DEFAULT_PARAMETERS_PATH
 
 from tkinter.filedialog import askopenfilename, askdirectory
 
 log = logging.getLogger(__name__)
 
+DEFAULT_PARAMETERS_PATH = 'bcipy/parameters/parameters.json'
 
-def _cast_parameters(parameters: dict) -> dict:
-    """Cast to Value.
 
-    Take in a parameters dictionary and coverts to a dictionary with type converted
-        and extranous information removed.
-    """
-    new_parameters = {}
-    for key, value in parameters.items():
-        new_parameters[key] = cast_value(value)
-
-    return new_parameters
-
-
-def cast_value(value):
-    """Cast Value.
+def copy_parameters(path: str = DEFAULT_PARAMETERS_PATH,
+                    destination: str = 'bcipy/parameters/') -> str:
+    """Creates a copy of the default configuration (parameters.json) to the
+    given directory and returns the path
 
-    Takes in a value with a desired type and attempts to cast it to that type.
+    Parameters:
+    -----------
+        destination: str - optional destination directory; default is the same
+          directory as the default parameters.
+    Returns:
+    --------
+        path to the new file.
     """
-    actual_value = str(value['value'])
-    actual_type = value['type']
-
-    try:
-        if actual_type == 'int':
-            new_value = int(actual_value)
-        elif actual_type == 'float':
-            new_value = float(actual_value)
-        elif actual_type == 'bool':
-            new_value = True if actual_value == 'true' else False
-        elif actual_type == 'str' or 'path' in actual_type:
-            new_value = str(actual_value)
-        else:
-            raise ValueError('Unrecognized value type')
-
-    except Exception:
-        raise ValueError(f'Could not cast {actual_value} to {actual_type}')
+    now = datetime.now()
+    month = str(now.month).rjust(2, "0")
```

Comment:
I think it's cleaner to use `strftime` for filenames with datetimes. I use it in `helpers/save`

Reply:
Thanks, I'll look at that.